packages/quantic/force-app/main/default/lwc/quanticGeneratedAnswerContent/quanticGeneratedAnswerContent.js

```diff
@@ -0,0 +1,62 @@
+import MARKED_JS from '@salesforce/resourceUrl/marked';
+import {transformMarkdownToHtml} from 'c/quanticUtils';
+// @ts-ignore
+import {loadScript} from 'lightning/platformResourceLoader';
+import {LightningElement, api} from 'lwc';
+// @ts-ignore
+import generatedMarkdownContentTemplate from './templates/generatedMarkdownContent.html';
+// @ts-ignore
+import generatedTextContentTemplate from './templates/generatedTextContent.html';
+
+export default class QuanticGeneratedAnswerContent extends LightningElement {
+  @api isStreaming;
+  @api answerContentFormat;
+  @api
+  get answer() {
+    return this._answer;
+  }
+  set answer(value) {
+    this._answer = value;
+    if (this.answerContentFormat === 'text/markdown') {
+      this.updateHtmlContent();
+    }
+  }
+
+  _ready = false;
+  _answer;
+
+  connectedCallback() {
+    if (this.answerContentFormat === 'text/markdown') {
+      loadScript(this, MARKED_JS + '/marked.min.js')
+        .then(() => {
+          this._ready = true;
+        })
+        .catch((error) => {
+          console.error('Error loading marked', error);
+        });
+    }
+  }
+
+  updateHtmlContent() {
+    const newHTMLContent =
+      // @ts-ignore
+      window.marked && transformMarkdownToHtml(this.answer, window.marked);
```

Comment:
:thinking: Suppose `window.marked` is falsy, then the element will be set to `false`... I guess, in this case we should raise an error.